lib_collections/models.py

```diff
@@ -604,7 +605,8 @@ def viewer(self, request, *args, **kwargs):
         InlinePanel(
             'supplementary_access_links', label='Supplementary Access Links'
         ),
-        InlinePanel('related_collection_placement', label='Related Collection'),
+        InlinePanel('related_collection_placement',
+                    label='Related Collection'),
```

Comment:
Just something to note, nothing we can do about it now but when I run `yapf` here (without anything else) it puts it on one line. I'm not saying you should do anything about this. I guess we just have to note it.